### ISSUE TITLE
PLOT: Line color turns red whenever the plot window is maximized

### ISSUE THREAD

**Issue by ShamrockLee** [gnudatalanguage/gdl#858]
**Description**

When maximizing the window containing the graph plotted using `PLOT` procedure, the color of the line and the edge turns red. Changing back to the original size doesn't change the color back.

**Environment setup**

Dockerfile
```Dockerfile
FROM rsolano/debian-slim-vnc
RUN apt-get update && apt-get install -y gnudatalanguage
```
`start.sh`
```start.sh
#!/usr/bin/env sh
docker build -t debian-gdl .
docker run --rm -v $PWD/config_dir:/home/debian/.config -v /dev/shm:/dev/shm -v $HOME/Documents:/home/debian/Documents -v $HOME/Projects:/home/debian/Projects -p 5900:5900 debian-gdl
```

**GDL version**

```
GDL> print, !gdl
{ 0.9.9 Apr 23 2019  1555977600}
```

**Reproduction steps**

1. Plot with the `PLOT` procedure with arbitrary `color` and `background` (say, `PLOT, [0, 1], [1, 0]`)
2. Maximize the plot window
3. Observe that the line and the edge of the plot turns red
4. Normalize the plot window
5. Observe that the color of the line and edge stays red

**Comment by alaingdl** [comment]
Hi. Yes it is a know issue. On the other side, doing the same, IDL don't change the color but does not re-scale ... Do we frequently re-scale the window in interactive mode ?!

For me, the absolute critical point is how fast is running the graphical output, on local but also through the network, e.g. running a large pipeline with multiple plots and images. We do have some tests on that, and thanks to the improvements made especially by Gilles, we are in line with IDL. No bottleneck here.

**Comment by ShamrockLee** [comment]
@alaingdl Thanks a lot for the reply.
It is a minor problem. I file this issue simply because I notice that.
Thanks you all for this amazing project!

**Comment by GillesDuvert** [comment]
solved in #882
